jersey-servlet-example-annotations/src/main/java/com/moesif/servlet/jersey/example/MoesifServletFilter.java

```diff
@@ -15,7 +15,7 @@
     urlPatterns ="/*",
     initParams = {
         @WebInitParam(name = "application-id",
-            value = "Your Moesif Application Id"),
+            value = "eyJhcHAiOiIxOTg6NzQ0IiwidmVyIjoiMi4xIiwib3JnIjoiNjQwOjEyOCIsImlhdCI6MTczMDQxOTIwMH0.duHFw2GFPGinL-RL_Bv2QyvxNzwO2oqtxnieO22P2B0"),
```

Comment:
We shouldn't check in keys. Technically our Application Ids are safe for untrusted apps but still.

Reply:
Was a hasty commit mistake at the end, removed